Scroller/EditableWrapper.js

```diff
@@ -374,6 +400,32 @@ const EditableWrapper = (props) => {
 		}
 	}, [removeItem, editable.removeItemFuncRef]);
 
+	useEffect(() => {
+		// addEventListener to moveItems while scrolled
+		const scrollContentNode = scrollContentRef.current;
+		const scrollContentCenter = scrollContentNode.getBoundingClientRect().width / 2;
+
+		const handleMoveItemsByScroll = () => {
+			const {centeredOffset, itemWidth, mouseClientX, selectedItem} = mutableRef.current;
+			if (selectedItem) {
+				const toIndex = Math.floor((mouseClientX + scrollContentNode.scrollLeft - centeredOffset) / itemWidth);
+				mutableRef.current.lastInputType = 'scroll';
+				if (mouseClientX > scrollContentCenter) {
+					moveItems(clamp(0, dataSize - 1, toIndex + 1));
+				} else {
+					moveItems(clamp(0, dataSize - 1, toIndex - 1));
+				}
+			}
+		};
+
+		scrollContentNode.addEventListener('scroll', handleMoveItemsByScroll);
+
+		return () => {
+			scrollContentNode.removeEventListener('scroll', handleMoveItemsByScroll);
+		};
+
+	}, [dataSize, moveItems, mutableRef, scrollContentRef]);
```

Comment:
I guess we could remove `dataSize` here.
And we don't need to add `mutableRef` here since it's from `useRef` it won't be changed by rerendering.